### ISSUE TITLE
Add back button for mobile

### ISSUE THREAD

**Issue by ben-z** [fredck/ha-data-editor#3]
On mobile, the editor takes up the entire screen and there's no way to go back. It would be awesome to have a back button, or keep the top bar visible.

![IMG_9335](https://github.com/fredck/ha-data-editor/assets/5977478/d5b88cd9-ea5e-4934-8d97-4d05ecc243a1)

**Comment by fredck** [comment]
Yes, that's messed up.

I'm wondering if there should be even support for mobile since this is a very technical integration.

Anyway, the UI aspect of this integration is pretty basic. I didn't investigate how to make it right since it essentially works. Anyone with experience on frontend coding would be welcome to join in an implement it the proper way.

**Comment by ben-z** [comment]
Makes sense! Is there a simple way to disable this on mobile? It’s easy to accidentally tap it 😄

**Comment by fredck** [comment]
For now, the way I do it is disabling the integration at the integrations page and re-enable it in the rare cases I need it. It's pretty fast to do it, it handles the mobile issue and, most importantly, it keep the sidebar less crowded.

But yes... mobile "support" is todo.

**Comment by ben-z** [comment]
Ah makes sense. Thanks, will do!
